### PR TITLE
update for SDXL Turbo support

### PR DESCRIPTION
**Pull request by atakaha**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: tests/test_diffusers.py [hunk 1]
```diff
@@ -933,6 +933,25 @@ def test_stable_diffusion_xl_euler_ancestral(self):
...
+
+        sd_pipe.set_progress_bar_config(disable=None)
+
+        inputs = self.get_dummy_inputs(device)
+        image = sd_pipe(**inputs).images[0]
+
+        image_slice = image[-3:, -3:, -1]
+
+        self.assertEqual(image.shape, (64, 64, 3))
+        expected_slice = np.array([0.4675, 0.5173, 0.4611, 0.4067, 0.5250, 0.4674, 0.5446, 0.5094, 0.4791])
```
Comment (regisss) [inline tests/test_diffusers.py:952]: These numbers were obtained on Gaudi1 or Gaudi2?

File: examples/stable-diffusion/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<!---
```
Comment (regisss) [inline examples/stable-diffusion/README.md:1]: This shouldn't be removed as it is a comment in the README so that it doesn't appear when it is rendered

Comment (atakaha) [reply examples/stable-diffusion/README.md]: fixed

**Review by regisss — APPROVED** [review]
LGTM!

**Pull request merged**